plugins/inputs/system/ps.go

```diff
@@ -99,21 +101,26 @@ func (s *systemPS) DiskUsage(
 			}
 		}
 
+		// If there's a host mount prefix, exclude other paths
+		if len(hostMountPrefix) > 0 && !strings.HasPrefix(p.Mountpoint, hostMountPrefix) {
```

Comment:
That doesn't make sense - if its point is to remove the prefix from paths, how can it *not* filter when there are conflicts?